src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_ILocalFunctionStatement.cs

```diff
@@ -256,13 +268,14 @@ void M()
     }
 }
 ";
-            string expectedOperationTree = @"
+string expectedOperationTree = @"
 ILocalFunctionStatement (Local Function: void Foo()) (OperationKind.LocalFunctionStatement) (Syntax: 'void Foo() => x++;')
   IBlockStatement (2 statements) (OperationKind.BlockStatement) (Syntax: '=> x++')
     IExpressionStatement (OperationKind.ExpressionStatement) (Syntax: 'x++')
       IIncrementExpression (UnaryOperandKind.IntegerPostfixIncrement) (OperationKind.IncrementExpression, Type: System.Int32) (Syntax: 'x++')
         Left: ILocalReferenceExpression: x (OperationKind.LocalReferenceExpression, Type: System.Int32) (Syntax: 'x')
     IReturnStatement (OperationKind.ReturnStatement) (Syntax: '=> x++')
+    null
```

Comment:
Is this `IReturnStatement.ReturnedValue`? If so, `OperationTreeVerifier.VisitReturnStatement` should probably visit the value explicitly with `Visit(operation.ReturnedValue, "Value");`.